database.yaml

```diff
@@ -1,7 +1,36 @@
 ---
 - name: Setup database
   hosts: database
+  become: yes
   tasks:
+    - name: update 
+      apt:
+        update_cache: yes
+    - name: upgrade
+      apt:
+        upgrade: dist
 
-    - name: message to remove
-      command: echo 'Here you gonna put all tasks and roles that gonna setup database'
+    - name: Install software 
+      apt:
+        name: 
+          - mysql-server
+     # DB will return an error if the db already exists, need to ignore errors 
+    - name: Create wordpress DB using sql file
+      shell: 'mysql < wordpress.sql'
+      ignore_errors: yes
```

Comment:
Not sure that I like `ignore_errors`. Instead maybe there is anything in ansible to work with MySQL?